### PR TITLE
[Bugfix:System] Capitalized Cypress System folder name

### PR DESCRIPTION
**Pull request by ZheyuDeng**
### What is the current behavior?
Inside the path Submitty\site\cypress\e2e, cypress system is not capitalized like other folders.
image
Current behavior:
Now cypress system is capitalized like others.
image

### CODE REVIEW

**Review by yanliw123 — APPROVED** [review]
image
The folder name now capitalized.

**Comment by IDzyre** [comment]
I need to wait for the Cypress test to run to make sure there is no conflict with the CI file and the capitalization, once they run, and if they pass, I will approve.

**Comment by bmcutler** [comment]
@ZheyuDeng 
Unfortunately this is failing cypress tests.

**Review by IDzyre — CHANGES_REQUESTED** [review]
In the .github/workflow/submitty_ci.yml file, change cypress-system to Cypress-System in all places also.

File: site/public/js/simple-grading.js [hunk 1]
```diff
@@ -252,6 +252,7 @@ function updateCheckpointCells(elems, scores, no_cookie) {
                     elem.attr('data-score', elem.data('score'));      // update the score
                     elem.attr('data-grader', elem.data('grader'));    // update new grader
                     elem.find('.simple-grade-grader').text(elem.data('grader'));
+
```
Comment (IDzyre) [inline site/public/js/simple-grading.js:255]: Are these changes relevant ?

Comment (ZheyuDeng) [reply site/public/js/simple-grading.js]: No it was from my another PR I have removed them.

**Pull request merged**